Firebase/Auth/Source/AuthProvider/OAuth/FIROAuthCredential.m

```diff
@@ -108,6 +120,7 @@ - (nullable instancetype)initWithCoder:(NSCoder *)aDecoder {
 
 - (void)encodeWithCoder:(NSCoder *)aCoder {
   [aCoder encodeObject:self.IDToken forKey:@"IDToken"];
+  [aCoder encodeObject:self.IDToken forKey:@"rawNonce"];
```

Comment:
Shouldn't this be `self.rawNonce` instead of `self.IDToken`?

Reply:
Thanks! Fixed here.

Reply:
Sorry, but where was it fixed? (I can't wait to use this!)

Reply:
https://github.com/firebase/firebase-ios-sdk/pull/4337

Reply:
Hey Michael, may I know how serious this bug is in your case? It helps us better understanding our developers. Thanks!

Reply:
I haven't used this yet, I was looking through the code because I plan to use this feature in the future.

Reply:
The fix will be in the next release. For now, the feature should be good enough to use.